src/App.tsx

```diff
@@ -15,36 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const CONST_SORT_VALUE = 'alph';
+const CONST_LENGTH_VALUE = 'length';
+
+interface GoodProps {
+  sortField: string;
+  reverse: boolean;
+}
```

Comment:
[Non critical] IMHO it is better to call it SortProps or something ike that. as GoodProps sound like these are some properties for each good (like quantity, category or etc.)